pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "ansys-product-library"
+authors = [{name = "ANSYS, Inc."}] # this is required
+
+# You can change the name. The email group works.
+maintainers = [{name = "PyAnsys developers", email = "pyansys.support@ansys.com"}]
+readme = "README.rst"
+description = "Template PyAnsys library"
+license = {file = "LICENSE"}
+
+# The version is automatically parsed from 'ansys/product/library/__init__.py'
+dynamic = ["version"]
+
+# Include all install requirements here.
+dependencies = []
+
+# Plan on supporting only the currently supported versions of Python
+requires-python = ">=3.7"
+
+# Less than critical but helpful
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+
+[project.optional-dependencies]
+docs = [
```

Comment:
The "docs", "style" and "test" requirements are now included here as optional dependencies. If you feel the approach with separate `requirements_*.txt` files is cleaner, I can revert this.

Reply:
I prefer having it all in one file, and that's partly a driver for using `pyproject.toml` (as well as being up with the times).